tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+    content: [],
+    theme: {
+        extend: {
+            colors: {
+                textfooter: '#70757a',
+                footerbg: '#f2f2f2;',
+                bordergrey: '#dadce0;',
+                linkbg: '#f8f9fa;',
+                linktext: '#3c4043',
+                languagecolor: '#1a0dab;'
+
```

Comment:
for the readability, the names of your custom style is not clear try to make the name that explain the color your are using.